bones/relationalBone.py

```diff
@@ -512,8 +512,9 @@ def _rewriteQuery(self, name, skel, dbFilter, rawFilter ):
 						v = db.Key( v )
 					dbFilter.ancestor( v )
 					continue
-				if not (k if "." not in k else k.split(".")[0]) in self.parentKeys:
-					logging.warning( "Invalid filtering! %s is not in parentKeys of RelationalBone %s!" % (k,name) )
+				boneName = k if "." not in k else k.split(".")[0]
```

Comment:
this can be replaced by just `boneName = k.split(".")[0]`

Reply:
Okay I have committed new changes according to your comment